XenAdmin/Dialogs/AttachUsbDialog.cs

```diff
@@ -72,6 +74,11 @@ private void BuildList()
         {
             Program.AssertOnEventThread();
 
+            labelWarningLine3.Visible = 
+                (platform == null) ||

```

Comment:
We can put this in a function in the VM class and reuse it in CreateVUSBAction.

Reply:
Good idea!